src/Razor/src/Microsoft.VisualStudio.RazorExtension/RazorPackage.cs

```diff
@@ -4,21 +4,32 @@
 using System;
 using System.Runtime.InteropServices;
 using System.Threading;
+using Microsoft.VisualStudio.LanguageServerClient.Razor;
 using Microsoft.VisualStudio.Shell;
 using Task = System.Threading.Tasks.Task;
 
 namespace Microsoft.VisualStudio.RazorExtension
 {
-    [PackageRegistration(UseManagedResourcesOnly = true, AllowsBackgroundLoading = true)]
+    // We attach to the 51st priority order because the traditional Web + XML editors have priority 50. We need to be loaded prior to them
+    // since we want to have the option to own the experience for Razor files
```

Comment:
51 -> 52???

Reply:
Ah, in my testing I upped it a single priority to verify how often pkgdefs get generated. I'll bring it back down to 51 below.